integration-tests/api-gateway/aws/authorization/authorization-happy.test.ts

```diff
@@ -0,0 +1,84 @@
+import { stackOutputs } from "../../../step-functions/aws/resources/cloudformation-helper";
+import {
+  clearAttemptsTable,
+  clearItemsFromTables,
+  getItemByKey,
+} from "../../../step-functions/aws/resources/dynamodb-helper";
```

Comment:
The resources folder should sit at the level of but step-functions, and api-gateway, you can leave this for now but a following PR should address this unless it is easier to do it here